### PR TITLE
refs #2 - manage context storage in workflow manager

### PR DESCRIPTION
**Pull request by adefretin**

### CODE REVIEW

**Review by jrmbrgs — APPROVED** [review]
it would be great to some test in key existence

**Comment by jrmbrgs** [comment]
merging to make disturb work again

File: Library/Services/ContextStorage.php [hunk 1]
```diff
@@ -68,47 +88,149 @@ public function __construct(Json $config)
...
      *
-     * @param string $key
+     * @param string $workflowProcessId
      *
      * @return bool
      */
-    public function exist(string $key) {
-        return $this->adpater->exist($key);
+    public function exist(string $workflowProcessId) {
+        return $this->adpater->exist($workflowProcessId);
```
Comment (ogarbe) [inline Library/Services/ContextStorage.php:109]: typo

Comment (jrmbrgs) [reply Library/Services/ContextStorage.php:109]: fixed in #57

**Pull request merged**